docker/gcp_pudl_etl.sh

```diff
@@ -133,9 +134,11 @@ function notify_slack() {
     message+="DISTRIBUTION_BUCKET_SUCCESS: $DISTRIBUTION_BUCKET_SUCCESS\n"
     message+="ZENODO_SUCCESS: $ZENODO_SUCCESS\n\n"
 
-    message+="See https://console.cloud.google.com/storage/browser/builds.catalyst.coop/$BUILD_ID for logs and outputs."
+    message+="Logs on <https://console.cloud.google.com/batch/jobsDetail/regions/us-west1/jobs/run-etl-$BUILD_ID/logs?project=catalyst-cooperative-pudl|Google Batch Console>.\n\n"
+    message+="See <https://console.cloud.google.com/storage/browser/builds.catalyst.coop/$BUILD_ID|GCS> for outputs."
 
     send_slack_msg "$message"
+    upload_file_to_slack "$LOGFILE" "pudl_etl logs for $BUILD_ID:"
```

Comment:
Do we want to keep this here? It's kind of nice to be able to click and download logs right from the failure notification. But we can already just click that Google Batch Console link...

Reply:
I guess what I'm saying with the links to logs & outputs is different people will prefer different access methods so maybe we should just provide them all.